lib/common.bash

```diff
@@ -281,3 +281,36 @@ restart_docker_service() {
 
 	return 0
 }
+
+# This function verifies that the dockerfile version is
+# equal to the test version in order to build the image or
+# just run the test
+check_dockerfiles_images()
+{
+	local image="$1"
+	local dockerfile_path="$2"
+
+	if [ -z "$image" ] || [ -z "$dockerfile_path" ]; then
+		die "Missing image or dockerfile path variable"
+	fi
+
+	# Verify that dockerfile version is equal to test version
+	check_image=$(docker images "$image" -q)
+	if [ -n "$check_image" ]; then
+		# Check image label
+		check_image_version=$(docker image inspect $image | grep -w DOCKERFILE_VERSION | head -1 | cut -d '"' -f4)
+		if [ -n "$check_image_version" ]; then
+			echo "$image is not updated"
+			build_dockerfile_image "$image" "$dockerfile_path"
+		else
+			# Check dockerfile label
+			dockerfile_version=$(grep DOCKERFILE_VERSION $dockerfile_path | cut -d '"' -f2)
```

Comment:
maybe grep for `DOCKERFILE_VERSION=` just to be sure :)

Reply:
actually maybe make it like `grep DOCKERFILE_VERSION= $dockerfile_path | cut -d '=' -f2 | tr -d \"` so it can deal with double-quoted (or not) version. 

```
[wmoschet@wainer-laptop tests]$ dockerfile_path="metrics/network/iperf3_dockerfile/Dockerfile"; grep DOCKERFILE_VERSION= $dockerfile_path | cut -d '=' -f2 | tr -d \"
1.0
[wmoschet@wainer-laptop tests]$ echo "DOCKERFILE_VERSION=1.0" | grep DOCKERFILE_VERSION= | cut -d '=' -f2 | tr -d \"
1.0

```

Reply:
@wainersm changes applied